### PR TITLE
Fix Search All Command Bookmark Seciton Returns opentabs instead of bookmark

### PR DESCRIPTION
**Pull request by reallinzc**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by a-laughlin** [comment]
Thanks for submitting a PR @reallinzc. It's very close.  There are a couple small things left:
- The code switches the data source to the bookmark data, but doesn't switch the component that the bookmark data relies on. That breaks the types.  See [how the bookmark search component does it](https://github.com/raycast/extensions/blob/8533f11972392b6d22f69f073fdb2af6d8ffee10/extensions/google-chrome/src/search-bookmarks.tsx#L26C10-L26C36).
- Make sure to `npm run fix-lint` before pushing. That'll catch issues like the types.
- Also, add your name to the contributors list in package.json.

File: extensions/google-chrome/src/search-all.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default function Command() {
         {bookmarkData.length === 0 ? (
           <List.Item title="No bookmarks found" key={"empty bookmark list item"} />
         ) : (
-          tabData.map((tab) => (
+          bookmarkData.map((tab) => (
             <ChromeListItems.TabList key={tab.key()} tab={tab} useOriginalFavicon={useOriginalFavicon} />
```
Comment (a-laughlin) [inline extensions/google-chrome/src/search-all.tsx:75]: Switching to bookmarkData is correct. Bookmark data relies on a different component though, hence typescript complaining about the "key" type on the TabList.  Both the bookmarkData and `ChromeListItems.TabList` need switching. See [how the bookmark search component does it](https://github.com/raycast/extensions/blob/8533f11972392b6d22f69f073fdb2af6d8ffee10/extensions/google-chrome/src/search-bookmarks.tsx#L26C10-L26C36).

Comment (a-laughlin) [reply extensions/google-chrome/src/search-all.tsx:75]: PR https://github.com/raycast/extensions/pull/15332

**Comment by pernielsentikaer** [comment]
Thanks for checking this @a-laughlin 🙏

@reallinzc could you check?

**Pull request closed without merging**